monkey/agent_plugins/credentials_collectors/chrome/src/windows_credentials_database_selector.py

```diff
@@ -1,10 +1,155 @@
-from pathlib import PurePath
-from typing import Sequence
+# NOTE: This only tries stealing credentials from the current user,
+#       so no DPAPI logic is implemented yet. That requires more research.
+#       We can decide to do this now or in a new version of the plugin.
+#       (WinAPI is used for current user, DPAPI is used otherwise.)
```

Comment:
I think that is fine, just create an issue for the enhancement so we can track it.